### ISSUE TITLE
Feedback: Update survey questions and timing of automation when service team's start creating release plans 

### ISSUE THREAD

**Issue by ladonnaq** [Azure/azure-sdk-tools#5463]
We have a release planner survey that is only for onboarding and survey link will be sent in the automated emails.  

- [ ] Update the survey questions and timing of automation when service team's start to use the release planner to create release plans 
- Link to the survey - https://aka.ms/azsdk/releaseplanner/feedback (Responses only) 
- Collaborator's link to the survey - https://forms.office.com/Pages/DesignPageV2.aspx?subpage=design&FormId=v4j5cvGGr0GRqy180BHbR-SRHOv9H51LkqMFimFYWbJUMlZaSVdNUzM5UkE1WjJYTlNXMlhMVkFPUy4u
- Email body for survey link - "Thanks for using our new Azure SDK Release Planner application! Your feedback matters to us, and we’d love to hear more about your experience using the Release Planner. 
Would you mind taking a quick 5-minute survey to help us improve the scheduler tool experience? 
Here is a link to the survey - https://aka.ms/azsdk/releaseplanner/feedback. 
Thank you in advance for your time. For any questions, please contact azsdkapex@microsoft.com."

**Comment by maririos** [comment]
I don't think there is already automation for the surveys when creating a release plan.
Could you expand more on the requirementes for this automated emails such as trigger/frequency, people to send it too, etc

**Comment by maririos** [comment]
Given the priority for the dashboard work, I am moving this issue out of this month.

@ladonnaq let me now if you don't agree.

**Comment by maririos** [comment]
@ladonnaq 
> Define when the survey will be sent

I am thinking, whenever the release plan is completed, to send the email.

**Comment by ladonnaq** [comment]
> @ladonnaq
> 
> > Define when the survey will be sent
> 
> I am thinking, whenever the release plan is completed, to send the email.

I agree with you. When a release plan is completed, send the release planner survey email.

**Comment by ladonnaq** [comment]
> After discussing with @ladonnaq , we decided to have a survey that will help us to know if users are getting value in using the Release planner apps.
> 
> Work to do:
> 
> * [ ]  Create/update survey. https://forms.office.com/Pages/DesignPageV2.aspx?subpage=design&id=v4j5cvGGr0GRqy180BHbR-SRHOv9H51LkqMFimFYWbJUREs5Slc3MldQM0s3WkhMUEs2VDlPWjRQVC4u
> * [ ]  Create aka.ms link
> * [ ]  Define when the survey will be sent
> * [ ]  Email body: "Thanks for using the Azure SDK Release Planner application. Your feedback matters to us, and we’d love to hear more about your experience using the Release Planner. Fill these less than 2 minute survey.
> 
> Thanks, The Azure SDK team"

Existing AKA Link for Release Planner Feedback - https://aka.ms/azsdk/releaseplanner/feedback (updated with the shortened url for the release planner feedback form).  New link for Onboarding feedback - https://aka.ms/azsdk/onboard/feedback (added short link to the onboard feedback form).

**Comment by maririos** [comment]
Because in production, the email templates that are sent after onboarding are using the `https://aka.ms/azsdk/releaseplanner/feedback` link, we need to be careful on when to update it.
As part of our next deployment we will start using `https://aka.ms/azsdk/onboard/feedback`, so for now I reverted it to point to `https://forms.office.com/r/FCecevKvzY`.

After deployment, we need to update `https://aka.ms/azsdk/onboard/feedback` to `https://forms.office.com/r/CaVh4Jf7wF`

**Comment by ladonnaq** [comment]
> Because in production, the email templates that are sent after onboarding are using the `https://aka.ms/azsdk/releaseplanner/feedback` link, we need to be careful on when to update it. As part of our next deployment we will start using `https://aka.ms/azsdk/onboard/feedback`, so for now I reverted it to point to `https://forms.office.com/r/FCecevKvzY`.
> 
> After deployment, we need to update `https://aka.ms/azsdk/onboard/feedback` to `https://forms.office.com/r/CaVh4Jf7wF`

@maririos Is this the only task left to complete before we close this GitHub issue?

**Comment by maririos** [comment]
> @maririos Mariana Rios Flores FTE Is this the only task left to complete before we close this GitHub issue?

@ladonnaq implementation of the email hasn't been done. After our discussion on Tuesday, it looks like you would prefer a UI change.

**Comment by ladonnaq** [comment]
The aka links to the surveys.  

1. https://aka.ms/azsdk/releaseplanner-typespec/feedback  
2. https://aka.ms/azsdk/releaseplanner/feedback

**Comment by maririos** [comment]
@ladonnaq do you still want automation for this? or given the lack of response we should keep doing it manually?

**Comment by ladonnaq** [comment]
FYI 

Mariana

 
@maririos - I added a link to the engagement experience/typespec survey to all of the APEX launch criteria descriptions in CLC.  Maybe I should start requiring survey completion before attestations are approved.  hahahaha  Just joking but I will start putting the survey link in every approved attestation to see if we can get more responses.  :-)

**Comment by ladonnaq** [comment]
@maririos I also posted an Announcement in the Onboarding channel 
<img width="440" alt="image" src="https://github.com/user-attachments/assets/a25331ca-f64d-466d-9ce4-b06ff06498f7">
Let me know if you think we should also post announcement in any other channels.  Perhaps the Azure SDK General, TypeSpec, ...
